tests/test-pluq.C

```diff
@@ -70,7 +70,7 @@ int main(int argc, char** argv){
     Field F(atof(argv[1]));
     Field::Element * A;
 
-    FFLAS::ReadMatrix (argv[2],F,m,n,A);
+    FFLAS::ReadMatrix (argv[2],F,m,n,A,FFLAS::FflasAuto);
```

Comment:
why did you need to change this?

Reply:
The implementation of ReadMatrix seems to have ever changed as the compiler raised error about this, I looked through the definition of ReadMatrix and adapted this line.